tools/swift-inspect/Sources/swift-inspect/Operations/DumpRawMetadata.swift

```diff
@@ -33,6 +57,18 @@ internal struct DumpRawMetadata: ParsableCommand {
       try process.context.allocations.forEach { allocation in
         let name: String = process.context.name(allocation: allocation.tag) ??  "<unknown>"
         print("Metadata allocation at: \(hex: allocation.ptr) size: \(allocation.size) tag: \(allocation.tag) (\(name))")
+        
+        if metadataOptions.summary {
+          if var allocatorTagTotal = allocatorTagTotals[Int(allocation.tag)] {
```

Comment:
There's a fun shorthand way to write this. Assuming this dictionary is now typed `[swift_metadata_allocation_tag_t: Int]` like I mentioned above, you can do this add-or-insert thing in a single line:

```
allocatorTagTotals[allocation.tag, default: 0] += allocation.size
```